api/client/amp.go

```diff
@@ -2,18 +2,16 @@ package client
 
 import (
 	"fmt"
-	"github.com/appcelerator/amp/api/rpc/logs"
-	"github.com/spf13/cobra"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/metadata"
 	"log"
-	"strconv"
 )
 
 const (
-	defaultPort   = ":50101"
-	serverAddress = "localhost" + defaultPort
+	defaultPort = ":50101"
+	// ServerAddress the server address
+	ServerAddress = "localhost" + defaultPort
```

Comment:
If this is a constant, then probably should be renamed to `DefaultServerAddress`.